app/Main.hs

```diff
@@ -61,10 +61,11 @@ makeCard c = do
     -- (mapM readMaybe) passes a list of strings to a list of Paradigms if possible. Since paradigms is NonEmpty not [] it must
     -- be converted to a list and then from a list again after
     paras <- fmap NE.fromList $ promptMult "\nEnter Card Paradigms:" 1 (mapM readMaybe) $ NE.toList . cardParadigms <$> c
-    prog <- promptMult "\nEnter Card Program:" 0 parseProg $ cardAction <$> c
+    act <- promptMult "\nEnter Card Program:" 0 parseProg $ cardAction <$> c
+    cost <- prompt "\nEnter Card Cost" $ cardCost <$> c
```

Comment:
I think you'll need a `promptInt` function for this, there's a nice one in the large arithmetic collide, and you should be able to adapt the default value bit pretty easily from the other prompt function

Reply:
Yeah, I was just implementing it!

Reply:
What is the default value exactly doing @oscar-h64 ?

Reply:
The default value if provided is so that when people are editing existing cards they can just leave the field blank if they don't want to change it

It's a maybe because that `makeCard` function is used by both create new card and edit card, and create new card doesn't have defaults

Reply:
Think ive got it